smoke-test/tests/cypress/cypress/e2e/incidentsV2/v2_incidents.js

```diff
@@ -168,4 +168,153 @@ describe("incidents", () => {
       );
     });
   });
+
+  it("Create V2 incident with all fields set and separate_siblings=false", () => {
+    cy.login();
+    cy.visit(
+      "/dataset/urn:li:dataset:(urn:li:dataPlatform:bigquery,cypress_project.jaffle_shop.customers,PROD)/Incidents?is_lineage_mode=false&separate_siblings=false",
+    );
+    cy.get('[data-testid="create-incident-btn-main"]').trigger("mouseover");
+    cy.get(".ant-dropdown-menu-item").first().click();
+    cy.get('[data-testid="incident-name-input"]').type(
```

Comment:
You can test here for a title --- You can verify that modal is open by verifying the title of modal - Create Incident